test/e2e/data/e2e_conf.yaml

```diff
@@ -12,12 +12,6 @@
 managementClusterName: capo-e2e
 
 images:
-- name: gcr.io/k8s-staging-cluster-api/cluster-api-controller:nightly_master_20210608
```

Comment:
Please keep those images and just change the tag to v0.4.0.
The idea is that the e2e tests download and import the cache into kind at the start of the e2e tests. This still makes sense with the v0.4.0 images

Reply:
Updated the tag to v0.4.0.